internal/request/graphql/schema/errors.go

```diff
@@ -182,3 +195,35 @@ func NewErrFieldTypeNotSpecified(objectName, fieldName string) error {
 		errors.NewKV("Field", fieldName),
 	)
 }
+
+func NewErrDirectiveWithUnknownArg(directive, arg string) error {
```

Comment:
todo: After dealing with my comment regarding the GQL validation, please review these and remove any that are no longer used.

Reply:
Done